pkgs/by-name/fl/flatpak/package.nix

```diff
@@ -1,84 +1,103 @@
 {
   lib,
   stdenv,
-  fetchurl,
-  fetchpatch,
-  runCommand,
   appstream,
-  autoreconfHook,
   bison,
   bubblewrap,
-  buildPackages,
   bzip2,
   coreutils,
   curl,
-  dbus,
   dconf,
   desktop-file-utils,
-  docbook_xml_dtd_45,
   docbook-xsl-nons,
+  docbook_xml_dtd_45,
+  fetchurl,
   fuse3,
+  gdk-pixbuf,
   gettext,
   glib,
   glib-networking,
   gobject-introspection,
   gpgme,
   gsettings-desktop-schemas,
-  gtk3,
   gtk-doc,
+  gtk3,
   hicolor-icon-theme,
-  intltool,
   json-glib,
   libarchive,
   libcap,
   librsvg,
   libseccomp,
+  libsoup_2_4,
   libxml2,
   libxslt,
+  malcontent,
+  meson,
+  ninja,
   nix-update-script,
-  nixosTests,
   nixos-icons,
   ostree,
   p11-kit,
   pkg-config,
   polkit,
-  pkgsCross,
   python3,
+  runCommand,
   shared-mime-info,
   socat,
   substituteAll,
   systemd,
   testers,
   valgrind,
-  which,
+  validatePkgConfig,
+  wayland,
+  wayland-protocols,
+  wayland-scanner,
   wrapGAppsNoGuiHook,
   xdg-dbus-proxy,
   xmlto,
   xorg,
-  xz,
   zstd,
+  withAutoSideloading ? false,
+  withDconf ? true,
+  withDocbookDocs ? true,
   withGtkDoc ? stdenv.buildPlatform.canExecute stdenv.hostPlatform,
+  withHttpBackend ? "curl",
+  withMalcontent ? true,
+  withMan ? true,
+  withSELinuxModule ? false,
+  withSeccomp ? true,
+  withSystemHelper ? true,
+  withSystemd ? true,
+  withWaylandSecurityContext ? true,
+  withXAuth ? true,
+  withZstd ? true,
```

Comment:
Are we sure we want that many new options? Why do we need them? Who tests variation of them?

Reply:
Fair

I've only left the doc options, selinux/systemd support (for any other systems/nixos derivatives that may ((not)) support them otherwise), and the auto sideloading feature. Most of these are simple toggles both here and internally, so I don't think they need much testing